client/src/app/site/motions/components/statute-paragraph-list/statute-paragraph-list.component.ts

```diff
@@ -0,0 +1,168 @@
+import { Component, OnInit } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+
+import { TranslateService } from '@ngx-translate/core';
+
+import { BaseComponent } from '../../../../base.component';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { PromptService } from '../../../../core/services/prompt.service';
+import { StatuteParagraph } from '../../../../shared/models/motions/statute-paragraph';
+import { ViewStatuteParagraph } from '../../models/view-statute-paragraph';
+import { StatuteParagraphRepositoryService } from '../../services/statute-paragraph-repository.service';
+import { EllipsisMenuItem } from '../../../../shared/components/head-bar/head-bar.component';
+
+/**
+ * List view for the statute paragraphs.
+ */
+@Component({
+    selector: 'os-statute-paragraph-list',
+    templateUrl: './statute-paragraph-list.component.html',
+    styleUrls: ['./statute-paragraph-list.component.scss']
+})
+export class StatuteParagraphListComponent extends BaseComponent implements OnInit {
+    /**
+     * content of the ellipsis menu
+     */
+    public menuList: EllipsisMenuItem[] = [
+        {
+            text: 'Sort statute paragraphs',
+            action: 'sortStatuteParagraphs'
+        }
+    ];
+
+    public statuteParagraphToCreate: StatuteParagraph | null;
+
+    /**
+     * Source of the Data
+     */
+    public statuteParagraphs: ViewStatuteParagraph[] = [];
+
+    /**
+     * The current focussed formgroup
+     */
+    public updateForm: FormGroup;
+
+    public createForm: FormGroup;
+
+    public openId: number | null;
+    public editId: number | null;
+
+    /**
+     * The usual component constructor
+     * @param titleService
+     * @param translate
+     * @param repo
+     * @param formBuilder
+     */
+    public constructor(
+        protected titleService: Title,
+        protected translate: TranslateService,
+        private repo: StatuteParagraphRepositoryService,
+        private formBuilder: FormBuilder,
+        private promptService: PromptService
+    ) {
+        super(titleService, translate);
+        const form = {
+            title: ['', Validators.required],
+            text: ['', Validators.required]
+        };
+        this.createForm = this.formBuilder.group(form);
+        this.updateForm = this.formBuilder.group(form);
+    }
+
+    /**
+     * Init function.
+     *
+     * Sets the title and gets/observes statute paragrpahs from DataStore
+     */
+    public ngOnInit(): void {
+        super.setTitle('Statute paragraphs');
+        this.repo.getViewModelListObservable().subscribe(newViewStatuteParagraphs => {
+            this.statuteParagraphs = newViewStatuteParagraphs;
+        });
+    }
+
+    /**
+     * Add a new Section.
+     */
+    public onPlusButton(): void {
+        if (!this.statuteParagraphToCreate) {
+            this.statuteParagraphToCreate = new StatuteParagraph();
+            this.createForm.setValue({
```

Comment:
After (successfully) sending the form "new" forms start in error state.

Clearing the values seems not sufficient to reset the form.
Try:
`this.createForm.reset();`

Reply:
Done.